src/main/java/frc/robot/JoysticksBindings.java

```diff
@@ -49,9 +49,11 @@ private static void mainJoystickButtons() {
         usedJoystick.POV_DOWN.whileTrue(Robot.swerve.getCommands().rotateToAngle(Rotation2d.fromDegrees(-17)));
 
         // Rotate Around FRONT_LEFT to 180 Deg
-        usedJoystick.POV_LEFT.whileTrue(Robot.swerve.getCommands().rotateToAngle(Rotation2d.fromDegrees(-17), RotateAxis.FRONT_LEFT_MODULE));
+        usedJoystick.POV_LEFT.whileTrue(Robot.swerve.getCommands()
```

Comment:
out of scope but, `Robot.swerve.getCommands()`, to DI the swerve to `JoysticksBindings`